apps/test/unit/lib/kits/maker/ui/SetupChecklistTest.js

```diff
@@ -48,7 +44,8 @@ describe('SetupChecklist', () => {
   });
 
   describe('test with expected console.error', () => {
-    allowConsoleErrors();
+    // Completely squelch console.error calls - ignore them and print nothing
+    beforeEach(() => console.error.reset());
```

Comment:
no no no, I only meant to revert this if the change was unintentional. it's fine to keep it in, I just thought it was something you didn't mean to commit. if you've already removed it, that's fine too.